comp/core/tagger/component.go

```diff
@@ -12,21 +12,30 @@
 package tagger
 
 import (
-	tagger_api "github.com/DataDog/datadog-agent/comp/core/tagger/api"
-	"github.com/DataDog/datadog-agent/comp/core/tagger/collectors"
+	"context"
+
 	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
+	taggertypes "github.com/DataDog/datadog-agent/pkg/tagger/types"
 	"github.com/DataDog/datadog-agent/pkg/tagset"
 )
 
 // team: container-integrations
 
 // Component is the component type.
 type Component interface {
-	Tag(entity string, cardinality collectors.TagCardinality) ([]string, error)
-	AccumulateTagsFor(entity string, cardinality collectors.TagCardinality, tb tagset.TagsAccumulator) error
+	Start(ctx context.Context) error
```

Comment:
I just saw your additional note 😓 then I think it's acceptable if we're gonna remove it later but you might want  thoughts from @hush-hush / @ogaca-dd

Reply:
Yeah, we try to move away from adding start and stop functions to the component interface. In this case I had to add them because it was part of the inner interface. I'm planning on removing on a separate PR

Reply:
Is there a card created somewhere to avoid forgetting about removing them? If no could you create it ?

Reply:
I will create one